### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by dosshra** [eblerjana/pangenie#8]
Hi
I started to install pangenie using the following code: 
`git clone https://github.com/eblerjana/pangenie.git`
`cd pangenie`
`conda activate pangenie`
`mkdir build; cd build; cmake .. ; make`
Since I use an old  linux HPC with old glibc I got this message:

> cmake: /lib64/libc.so.6: version `GLIBC_2.17' not found

I installed glibc2.17 using this code:
`conda install -c fridh glibc`
Then I reactivated pangenie and tried 
`cmake`
at the pangenie directory.
I get: 

> Segmentation fault

Thank you

**Comment by eblerjana** [comment]
I don't think this problem (and the segfault) is related to PanGenie, but is rather related to the old glib version installed on the system. Did you try to install it inside of a conda environment as explained in the README? Because I noticed that you missed the step that actually creates the environment when you listed your commands above: `conda env create -f environment.yml` (see README).
